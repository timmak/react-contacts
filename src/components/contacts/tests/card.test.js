import React from "react";
import renderer from "react-test-renderer";
import Card from "./../card";

it("should match snapshot", () => {
  const card = renderer.create(
    <Card firstName="John" lastName="Snow" email="john@thewall.com" />
  );
  const tree = card.toJSON();
  expect(tree).toMatchSnapshot();
});
