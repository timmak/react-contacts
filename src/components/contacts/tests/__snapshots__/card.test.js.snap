// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="contact-card"
>
  <dl>
    <dt
      className="contact-card--title"
    >
      Name
    </dt>
    <dd
      className="contact-card--value"
    >
      John Snow
    </dd>
    <dt
      className="contact-card--title"
    >
      Email
    </dt>
    <dd
      className="contact-card--value"
    >
      john@thewall.com
    </dd>
  </dl>
  <button
    className="button"
    href="/contact/edit/john@thewall.com"
    onClick={[Function]}
  >
    Edit Contact
  </button>
  <button
    className="button"
    onClick={undefined}
  >
    Delete Contact
  </button>
</div>
`;
